hawc/apps/myuser/api.py

```diff
@@ -25,4 +26,6 @@ class HawcValidateAuthToken(APIView):
     permission_classes = [IsAuthenticated]
 
     def get(self, request, format=None):
+        # create a django session for django view
+        login(request, request.user)
```

Comment:
@munnsmunns fyi, I simplified to just log the user in, since they're already confirmed to be authenticated